ceci/sites/local.py

```diff
@@ -79,11 +79,19 @@ def configure_for_parsl(self):
     def configure_for_mini(self):
         import psutil
 
-        cores = psutil.cpu_count(logical=False)
-        cores = self.config.get("max_threads", cores)
+        # The default is to allow a single process
+        # with as many cores as possible, but allow the
+        # user to specify both max_processes and max_threads
+        # to customize
+        procs = self.config.get("max_processes", 1)
+        cores_available = psutil.cpu_count(logical=False)
+        threads_default = max(cores_available // procs, 1)
+        cores = self.config.get("max_threads", threads_default)
```

Comment:
So you can still try to use `max_threads` to try and assign more cores to each process than available. I suppose that is ok?

Reply:
right - yes, for debugging or whatever the user wants to try.  Especially as for some systems you might want to use more threads than cores, depending on the architecture.

Reply:
If that make sense are you happy for me to merge?

Reply:
yes, sounds good then, just wanted to make sure it was intended

Reply:
Thanks - will merge!